lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -371,6 +371,16 @@ $rich-list-item-background-colors: (
 				padding: 0 $space-xxxxs;
 			}
 		}
+
+		#{$root}__metaData {
+			padding-left: $space-xxs;
+			padding-right: $space-xxxxs;
+
+			@media #{breakpoint-s()} {
+				justify-content: flex-start;
+				padding-left: 0;
```

Comment:
meData slot wrapper has 4ox left margin for small size and above mobile layout

Reply:
it's funny because Ewa said:
> :x:metaData - w rozmiarze M - ma lewy:12px, prawy: 6px, a powinien mieć 0px/4px
> :x:metaData - w rozmiarze S - ma lewy:12px, prawy: 6px, a powinien mieć 0px/4px